hail_scripts/v02/utils/elasticsearch_client.py

```diff
@@ -204,4 +204,4 @@ def export_table_to_elasticsearch(
         es.batch.write.refresh // default true  (Whether to invoke an index refresh or not after a bulk update has been completed)
         """
 
-        self.es.indices.forcemerge(index=index_name)
+        self.es.indices.forcemerge(index=index_name, request_timeout=60)
```

Comment:
I think the value is in seconds so this won't change the current 1m config. https://elasticsearch-py.readthedocs.io/en/7.10.0/api.html#timeout